array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,16 @@
+/* exported initial */
+
+// dec func 
+// Find the length of array arg minus 1, assgin it to a var
+// Remove last index 
+// return remaining items 
+
+function initial(array) {
+  var lastIndex = array.length - 1;
+  delete array[lastIndex];
```

Comment:
Oh no, you don't want to do this, this will change the original array. If this function was called, it would be because the caller wants to see the array minus the last element, not that they want to delete that value from the original array.